core/build.gradle

```diff
@@ -77,6 +77,17 @@ def fragileTestPatterns = [
     // Currently changes a global configuration parameter that for some reason
     // results in timestamp inversions for other tests.  TODO(mmuller): fix.
     "google/registry/flows/host/HostInfoFlowTest.*",
+
+    // Tests that have started exhibiting flakiness (IllegalStateException in
+    // JpaTransactionManager.  JPA apparently stops thinking we're in a
+    // transacti
+    "google/registry/tools/UnlockDomainCommandTest.*",
```

Comment:
This seems to just be a randomly selected collection of tests to me, i.e. it's not really going to do anything unless you add MOST tests to fragile (and then the test suite will take forever to run).